environment-dev.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
 - cf_xarray >=0.10.0 # not sure if we need to be that strict
 - cftime >=1.4.1
 - dask >=2024.8.1
+- importlib-metadata < 8.7.0
```

Comment:
Do we know where the issue is arising from? It looks like `dask` isn't configured properly. Any related issues?